ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,327 @@
+"""
+Merge DL1-files from stage1-process tool
+"""
+from pathlib import Path
+import sys
+import tables
+import os
+from argparse import ArgumentParser
+import numpy as np
+
+from tqdm import tqdm
+
+from ..core import Provenance
+from ctapipe.core import Tool, traits
+from traitlets import List
+from ctapipe.instrument import SubarrayDescription
+
+import warnings
+
+warnings.filterwarnings('ignore', category=tables.NaturalNameWarning)
```

Comment:
Why is this needed? Why are those warnings created?

Reply:
There is a NaturalNameWarning for `/dl1/service/image_statistics.__table_column_meta__` :
```
/home/jonas/anaconda3/envs/ma/lib/python3.7/site-packages/tables/path.py:155: NaturalNameWarning: object name is not a valid Python identifier: 'image_statistics.__table_column_meta__'; it does not match the pattern ``^[a-zA-Z_][a-zA-Z0-9_]*$``; you will not be able to use natural naming to access this object; using ``getattr()`` will still work, though
  check_attribute_name(name)
```
So you can't reach it just by `file.root.dl1.service.image_statistics.__table_column_meta_` but with `file.root['/dl1/service/image_statistics.__table_column_meta_']` still works. For the way it merges, its not important i think.